Makefile

```diff
@@ -31,6 +31,15 @@ all: build
 clean:
 	go clean -v $(GO_DEPS)
 
+kube-delete:
```

Comment:
These are such common commands that it doesn't make a lot of sense to put them in the Makefile. Users would have to learn the make targets defined here, but since they using Kubernetes, they almost certainly know how to use kubectl already. Besides, we document all of these commands in the readme.